test/e2e/framework/pod/resource.go

```diff
@@ -36,6 +36,7 @@ import (
 	podutil "k8s.io/kubernetes/pkg/api/v1/pod"
 	"k8s.io/kubernetes/pkg/client/conditions"
 	kubepod "k8s.io/kubernetes/pkg/kubelet/pod"
+	e2efwk "k8s.io/kubernetes/test/e2e/framework"
```

Comment:
This is pretty much an abomination as far as I'm concerned. If every import needs its own name... then you're doing a TERRIBLE job naming your packages. I'm going to stop approving PRs that do this.

/assign @timothysc

Reply:
fwiw I don't think `framework` needs an import alias, we only end up import aliasing packages like `test/e2e/framework/node` that is likely to conflict with other packages.

Reply:
the only reason to import alias framework libraries is the collision with existing names.  

Framework itself should not need this.